contents/blog/5-ways-to-improve-analytics-data.md

```diff
@@ -0,0 +1,133 @@
+---
+date: 2023-09-13
+title: 5 ways to improve your analytics data
```

Comment:
Thanks both for the suggestions. I went back and forth on this a bunch of times. In the end I think "5 ways to improve your product analytics data" is the most eye catching
```suggestion
title: 5 ways to improve your product analytics data
```